### PR TITLE
Fixes teshari loneliness for plushies, also removes thermal vision and short sightedness

### PR DESCRIPTION
**Pull request by Dropshotz**
Title.
Plushies now accounted for loneliness mechanic, calm teshari down
Removed thermals, and short sightedness. Both I feel are unnecessary.

### CODE REVIEW

**Comment by cebutris** [comment]
The plushy thing I can understand since we don't have the pop.

But the thermal/shortsight can stay tbh. It's nice flavor and helps bring them away from "Human but different"

**Comment by Dropshotz** [comment]
Should I make a request that is only for the plushies?

**Comment by TalkingCactus** [comment]
> Should I make a request that is only for the plushies?

yes

**Comment by Dropshotz** [comment]
Changed it to only affect plushies now

File: code/modules/mob/living/carbon/human/species/station/seromi.dm [hunk 1]
```diff
@@ -56,7 +56,7 @@
 	mob_size = MOB_SMALL
 	pass_flags = PASSTABLE
 	holder_type = /obj/item/weapon/holder/human
-	short_sighted = 1
+	short_sighted = 0
```
Comment (TalkingCactus) [inline code/modules/mob/living/carbon/human/species/station/seromi.dm]: a

Comment (Dropshotz) [reply code/modules/mob/living/carbon/human/species/station/seromi.dm]: did it fuck up. Sorry.

Comment (Dropshotz) [reply code/modules/mob/living/carbon/human/species/station/seromi.dm]: https://cdn.discordapp.com/attachments/729321033475293204/733244394538270750/unknown.png

no it didn't? I reverted those changes

Comment (TalkingCactus) [reply code/modules/mob/living/carbon/human/species/station/seromi.dm]: git is flawless

Comment (Dropshotz) [reply code/modules/mob/living/carbon/human/species/station/seromi.dm]: I have to agree on that one

**Pull request merged**